.gitignore

```diff
@@ -4,3 +4,4 @@
 *.o
 /vis
 /dependency
+.DS_Store
```

Comment:
Generally a good idea to put systems specific ignores in your global git ignore (`~/.gitignore`).